src/pages/register.tsx

```diff
@@ -0,0 +1,122 @@
+  const [displayName, setDisplayName] = React.useState<string>("");
+  const [gender, setGender] = React.useState<string>("");
+
+  const handleDisplayNameChange = (
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void => {
+    setDisplayName(event.target.value);
+  };
+
+  function handleEmailChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setEmail(event.target.value);
+  }
+
+  function handlePasswordChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setPassword(event.target.value);
+  }
+
+  function handleConfirmPasswordChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setConfirmPassword(event.target.value);
+  }
+
+  function handleGenderChange(
+    event: SelectChangeEvent<string>
+  ): void {
+    setGender(event.target.value)
+  }
+
+  return (
+    <Stack spacing={3} alignItems="center" justifyContent="center" style={{ minHeight: "100vh" }}>
+      <Logo width={119} height={119} />
+
+      <Box style={{ width: "50vw" }}>
+        <CommonTextField
+          header="Username"
+          placeholder="Display Name"
+          value={displayName}
+          handleValueChange={handleDisplayNameChange}
+          char_limit={100}
+          isErr={false}
+          errMsg=""
+        />
+      </Box>
+
+      <Box style={{ width: "50vw", marginTop: 0 }}>
+        <CommonTextField
+          header="Email"
+          placeholder="Email"
+          value={email}
+          handleValueChange={handleEmailChange}
+          isErr={false}
+          errMsg=""
+        />
+      </Box>
+
+      <Box style={{ width: "50vw" }}>
+        <PasswordTextFeild
+          header="Password"
+          placeholder="Password"
+          value={password}
+          handleValueChange={handlePasswordChange}
+          isErr={false}
+          errMsg=""
+        />
+      </Box>
+
+      <Box style={{ width: "50vw" }}>
+        <PasswordTextFeild
+          header="Confirm Password"
+          placeholder="Confirm Password"
+          value={confirmPassword}
+          handleValueChange={handleConfirmPasswordChange}
+          isErr={false}
+          errMsg=""
+        />
+      </Box>
+
+      <Typography>Gender</Typography>
+      <Select sx={{ width: "100px" }} onChange={handleGenderChange} defaultValue="">
+        {(Object.keys(Gender) as (keyof typeof Gender)[]).map((key, index) => (
+          <MenuItem value={key} key={index}>{Gender[key]}</MenuItem>
+        ))}
+      </Select>
+
+      <Typography>Birth Date</Typography>
+      <LocalizationProvider dateAdapter={AdapterDayjs}>
+        <DatePicker
+          label="Birth Date"
+          value={value}
+          onChange={(newValue) => {
+            setValue(newValue);
+          }}
+          renderInput={(params) => <TextField {...params} />}
+        />
+      </LocalizationProvider>
```

Comment:
I don't think this is the same design as in figma. Please correct it by comparing with tum's code.

Reply:
Lgtm. However, the box size still weird. It's look bigger than the others.